internal/repository/filesystem.go

```diff
@@ -465,12 +466,20 @@ func CreateConfigDir() (string, error) {
 }
 
 func makeDir(dir string) error {
-	if _, err := os.Stat(dir); os.IsNotExist(err) {
-		if err := os.Mkdir(dir, 0755); err != nil {
-			return err
+	dir = strings.ReplaceAll(dir, "\\", "/")
```

Comment:
Lets use file path method to normalise the path. 
```suggestion
	dir = filepath.FromSlash(dir)
```